docker-compose.yml

```diff
@@ -41,6 +41,7 @@ services:
       MYSQL_HISTFILE: "/home/solidus_user/history/mysql_history"
       RAILS_ENV: development
       ACTIVE_STORAGE_VARIANT_PROCESSOR: "vips"
+      SANDBOX_HOST: "0.0.0.0"
```

Comment:
Looking at rails core I think you can just provide a BINDING env variable which will be used if no option is provided

https://github.com/rails/rails/blob/654704247eba742e139cfaa8d1385f13605d9e12/railties/lib/rails/commands/server/server_command.rb#L156-L158

Reply:
Oh, great call! Done :slightly_smiling_face: